content/en/account_management/rbac/permissions.md

```diff
@@ -0,0 +1,372 @@
+
+##### logs_modify_indexes
+
+Grants a role the ability to create and modify log indexes. This includes:
+
+- Setting inclusion queries for which logs should be routed into an index.
+- Setting log retention for an index.
+- Limiting which roles have read access on an index (`logs_read_index_data`).
+- Which roles can modify exclusion filters for an index (`logs_write_exclusion_filters`).
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_modify_indexes_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_exclusion_filters
+
+Grants a role the ability to create or modify exclusion filters within an index. This can be assigned either globally or restricted to a subset of indexes.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+**Global access**:
+
+Go to your [Datadog Roles Page][1] and select the checkbox `write` as below for the wanted role:
+
+{{< img src="account_management/rbac/logs_write_exclusion_filters_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+**Subset of indexes**:
+
+1. Remove the global permission on the role.
+2. Grant this permission to the role in [the Processing Pipelines page of the Datadog app][2] by editing an index and adding a role to the "Grant editing Exclusion Filters of this index to" field (screenshot below).
+
+{{< img src="account_management/rbac/logs_write_exclusion_filters.png" alt="Grant write access on index exclusion filters to specific roles"  style="width:75%;" >}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+[2]: https://app.datadoghq.com/logs/pipelines
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_pipelines
+
+Grants a role the ability to create and modify log processing pipelines. This includes setting matching filters for what logs should enter the processing pipeline, setting the name of the pipeline, and limiting which roles have write access on the processors within that pipeline (`logs_write_processors`).
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_write_pipeline_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+To grant write access to only two processing pipelines whose IDs are `abcd-1234` and `bcde-2345` respectively:
```

Comment:
where do people find the pipeline IDs? it'd be nice if we can include that info.

Reply:
That's explain in the API documentation. The goal was just to illustrate it here but not to re-do the whole API workflow.
But feel free to make a suggestion if you believe it can help.